Makefile

```diff
@@ -18,13 +19,13 @@ sanitize:
 	hooks/run_vet.sh
 
 test-unit: clean deps sanitize build
-	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go test --test.short github.com/GoogleCloudPlatform/heapster/...
+	GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go test --test.short github.com/GoogleCloudPlatform/heapster/... $(FLAGS)
 
 test-unit-cov: clean deps sanitize build
 	hooks/coverage.sh
 
-test-integration: clean deps
-	godep go test -v --timeout=30m github.com/GoogleCloudPlatform/heapster/integration/... --vmodule=*=2 
+test-integration: clean deps build
+	godep go test -v --timeout=30m github.com/GoogleCloudPlatform/heapster/integration/... --vmodule=*=2 $(FLAGS)
```

Comment:
Its for developer use.

On Fri, Jun 12, 2015 at 3:03 PM, Rohit Jnagal <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/GoogleCloudPlatform/heapster/pull/335#discussion_r32359189>
> :
>
> >
> >  test-unit-cov: clean deps sanitize build
> >  	hooks/coverage.sh
> >
> > -test-integration: clean deps
> > -	godep go test -v --timeout=30m github.com/GoogleCloudPlatform/heapster/integration/... --vmodule=*=2
> > +test-integration: clean deps build
> > +	godep go test -v --timeout=30m github.com/GoogleCloudPlatform/heapster/integration/... --vmodule=*=2 $(FLAGS)
>
> where do we fill the FLAGS? I don't see that anywhere.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/heapster/pull/335/files#r32359189>
> .
>